packages/definitions/dist/fhir/r4/fhir.schema.json

```diff
@@ -59503,7 +59503,8 @@
               "graphql-introspection",
               "terminology",
               "websocket-subscriptions",
-              "reference-lookups"
+              "reference-lookups",
+              "transactions"
```

Comment:
How would you feel about renaming this to `transaction-bundles`?

We will use `transactions` for conditional create, conditional update.  This feature flag is just for the `transaction` bundles.